ndk/src/font.rs

```diff
@@ -0,0 +1,397 @@
+//! Bindings for [`AFont`], [`AFontMatcher`], and [`ASystemFontIterator`]
+//!
+//! [`AFont`]: https://developer.android.com/ndk/reference/group/font
+//! [`AFontMatcher`]: https://developer.android.com/ndk/reference/group/font#afontmatcher_create
+//! [`ASystemFontIterator`]: https://developer.android.com/ndk/reference/group/font#asystemfontiterator_open
+
+use std::convert::TryFrom;
+use std::ffi::{CStr, OsStr};
+use std::fmt::{self, Write};
+use std::os::unix::prelude::OsStrExt;
+use std::path::Path;
+use std::ptr::NonNull;
+use std::result::Result;
```

Comment:
I suggested to drop the qualification because:

1. You didn't use it in all places;
2. It is in scope by default through Rust's prelude: https://doc.rust-lang.org/std/prelude/index.html.

```suggestion
```

Reply:
I forgot that `Result` is included in the prelude. Actually, I use the full path when it comes to `Result` or `Error` because many crates have types with the same name... Fixed!